clients/src/test/java/org/apache/kafka/common/requests/ProduceRequestTest.java

```diff
@@ -125,7 +126,8 @@ public void testBuildWithCurrentMessageFormat() {
                                         new ProduceRequestData.PartitionProduceData().setIndex(9).setRecords(builder.build()))))
                                 .iterator()))
                         .setAcks((short) 1)
-                        .setTimeoutMs(5000));
```

Comment:
total nit: the spacing here (and a few other tests) looks like 6 spaces. Could we make it 4?